uclapi.openapi.json

```diff
@@ -0,0 +1,4256 @@
+                "description": "Longitude of the location of the site."
+              }
+            }
+          },
+          "free_seats" : {
+            "type" : "string",
+            "example" : "28",
+            "description": "Number of free seats in the room."
+          }
+        }
+      },
+      "survey" : {
+        "properties" : {
+          "id" : {
+            "type" : "number",
+            "example" : 46,
+            "description": "The library survey's ID. This is used later to identify individual libraries."
+          },
+          "name" : {
+            "type" : "string",
+            "example" : "UCL Institute of Education Library",
+            "description": "The human-readable name of the library."
+          },
+          "active" : {
+            "type" : "boolean",
+            "example" : true,
+            "description": "Whether the survey is active and working."
+          },
+          "start_time" : {
+            "type" : "string",
+            "example" : "09:00",
+            "description": "Standard UCL opening time supplied by the backend. This field should NOT be trusted as information on when a particular library is open. This is usually set to 09:00."
+          },
+          "end_time" : {
+            "type" : "string",
+            "example" : "17:00",
+            "description": "Standard UCL closing time supplied by the backend. This field should NOT be trusted as information on when a particular library closes. This is usually set to 17:00."
+          },
+          "staff_survey" : {
+            "type" : "number",
+            "example" : 0,
+            "description": "Whether the survey represents a staff workspace (`true`) or a student work or study space (`false`). This is useful in apps where you set the `survey_filter` parameter to `all` and wish to do filtering of student and staff workspaces in your app or API as opposed to leaving the filtering to UCL API. By default, as `survey_filter` is set to `student` unless you specify otherwise, all workspaces will have this parameter set to `false`."
+          },
+          "maps" : {
+            "type" : "array",
+            "description": "List of maps corresponding to regions within the library's survey.",
+            "items" : {
+              "$ref" : "#/components/schemas/map"
+            }
+          },
+          "location" : {
+            "type" : "object",
+            "properties" : {
+              "coordinates" : {
+                "type" : "object",
+                "description": "Object containing a latitude and longitude.",
+                "properties" : {
+                  "lat" : {
+                    "type" : "string",
+                    "example" : "51.522897",
+                    "description": "Latitude of the survey location."
+                  },
+                  "lng" : {
+                    "type" : "string",
+                    "example" : "-0.127864",
```

Comment:
nit: number in quotes

Reply:
Turns out we actually return lat/long as a string! that seems intentional, maybe for storage or precisions concerns?